### PR TITLE
Pluggable deploy as, download as mechanism

### PR DESCRIPTION
**Pull request by parente**
For issue #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +37,56 @@ If you want to try the extension and demos without installing it yourself, visit
...
+  '''
+  Transforms, converts, bundles, etc. the notebook. Then issues a Tornado web 
+  response using the handler to redirect the browser, download a file, show
+  an HTML page, etc. This function must finish the handler response before
+  returning either explicitly or by raising an exception.
+
+  :param handler: The tornado web handler that serviced the request
+  :param absolute_notebook_path: The path of the notebook on disk
+  '''
+  self.finish('Hello world!'')
```
Comment (parente) [inline README.md]: Should be called on handler, not self.

File: README.md [hunk 2]
```diff
@@ -36,6 +37,56 @@ If you want to try the extension and demos without installing it yourself, visit
...
+```python
+from tornado import gen
+
+@gen.coroutine
+def bundle(handler, absolute_notebook_path):
+  # simulate a long running IO op (e.g., deploying to a remote host)
+  yield gen.sleep(10)
+
+  # now respond
+  self.finish('I slept for 10 seconds!')
```
Comment (parente) [inline README.md]: handler, self

**Comment by jtyberg** [comment]
Reviewed with @parente.  This makes sense to me to let other extensions bring in other bundlers.

**Comment by parente** [comment]
Merging this in to pave the way for the other initial bundler impls to go in as well.

**Pull request merged**